### PR TITLE
Request to adding base Java classes architecture

### PR DESCRIPTION
**Pull request by ghost**
- Class structure according to UML was created  752f72a
- Little fix in UML  802706b
- Some fixes in base Java classes architecture  7db992f
- Small fix in UML diagram  dba3633

### CODE REVIEW

File: src/main/java/rape/brutal/gitstarship/Storage.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+package rape.brutal.gitstarship;
+
+import java.util.HashMap;
+
+/**
+ * Created by haze on 22.03.2017.
+ */
+public class Storage {
+        private HashMap<String, IStorageable> items;
```
Comment (ViRGiL175) [inline src/main/java/rape/brutal/gitstarship/Storage.java]: Bad naming

**Review by ViRGiL175 — CHANGES_REQUESTED** [review]
- Shitty code arranging
- Access modifiers errors
- Dirty interfaces
- Getters and setters needed

**Comment by ViRGiL175** [comment]
Reviewed.

File: src/main/java/rape/brutal/gitstarship/Storage.java (hunk 1 above)
Comment (ghost) [reply src/main/java/rape/brutal/gitstarship/Storage.java]: All fixed!

File: src/main/java/rape/brutal/gitstarship/locations/Location.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+package rape.brutal.gitstarship.locations;
+
+import rape.brutal.gitstarship.Storage;
+import rape.brutal.gitstarship.ships.GitStarship;
+
+/**
+ * Created by haze on 22.03.2017.
+ */
+public abstract class Location {
```
Comment (ViRGiL175) [inline src/main/java/rape/brutal/gitstarship/locations/Location.java:14]: Line gap was lost.

**Review by ViRGiL175 — CHANGES_REQUESTED** [review]

**Comment by ViRGiL175** [comment]
Reviewed.

**Pull request merged**